test/test.js

```diff
@@ -1,35 +1,35 @@
 var assert = require( 'assert' );
 var rollup = require( 'rollup' );
 var md = require( '../dist/rollup-plugin-md.js' );
-var npm = require( 'rollup-plugin-node-resolve' );
+var npm = require( '@rollup/plugin-node-resolve' );
 
 require( 'source-map-support' ).install();
 
 process.chdir( __dirname );
 
 function executeBundle ( bundle ) {
-    var generated = bundle.generate();
-    var code = generated.code;
+	var generated = bundle.generate();
+	var code = generated.code;
 
-    var fn = new Function( 'assert', code );
-    fn( assert );
+	var fn = new Function( 'assert', code );
+	fn( assert );
 }
 
 describe( 'rollup-plugin-md', function () {
-    it( 'converts md', function () {
-        return rollup.rollup({
-            entry: 'samples/main.js',
-            plugins: [ md({
-                marked: {
-                      gfm: true,
-                      tables: true,
-                      breaks: false,
-                      pedantic: false,
-                      sanitize: true,
-                      smartLists: true,
-                      smartypants: false
-                }
-            }) ]
-        }).then( executeBundle );
-    });
+	it( 'converts md', function () {
+		return rollup.rollup({
+			input: 'samples/main.js',
+			plugins: [ md({
+				marked: {
+					gfm: true,
+					tables: true,
+					breaks: false,
+					pedantic: false,
+					sanitize: true,
+					smartLists: true,
+					smartypants: false
+				}
+			}) ]
+		}).then( executeBundle );
+	});
 });
```

Comment:
The code change here is on line 21: 'entry' => 'input' to match the current rollup API. The rest is indentation.

I am not a fan of tabs, but since you set up the .eslint rules to call for it, I invoked the rules, so you would not have mixed indentation.